src/cli.ts

```diff
@@ -61,6 +66,19 @@ if (configResults) {
   Object.assign(options, configResults.config)
 }
 
+if (cli.flags.exportConfig) {
+  if (options.typescript) {
+    const config = getTSLintConfig(options)
+    fs.writeFileSync('./tslint.json', JSON.stringify(config, null, 2))
+    console.log('tslint.json generated')
```

Comment:
nitpick: Instead of just a normal log, could you change it to:

```ts
console.log(chalk.bold.green('\n\u2714 tslint.json generated\n'))
```

just to be a bit more consistent with the rest of lynt's output format